### ISSUE TITLE
Shimmer container is rendered even with null / false children

### ISSUE THREAD

**Issue by Zeikko** [tomzaku/react-native-shimmer-placeholder#58]
I'm experiencing a situation where the shimmer is container View is rendered even when the children is null / false.
I expected the shimmer container view not to be rendered when children is null / false.
This causes the shimmer to reserve space according to its height and width even when there is nothing to render.

Example JSX:
```
<ShimmerPlaceholder visible={!loading}>
  {Boolean(data) && <Text>{data}</Text>}
</ShimmerPlaceholder>
```
In case where `loading` is `false` but data is falsy, this JSX render an empty View. I think it would be better to not render anything in this case. I think this could be changed by adding the following code to the `BaseShimmerPlaceholder`
```
if (visible && (children === null || children === false)) {
  return null
}
```
Is this an issue or a design decision that has been made? Is my proposed fix feasible or are there some issues with it?

**Comment by tomzaku** [comment]
Interesting. But this is the edge case. Let's see if anyone really needs this.
As I know you can handle it from your side right

**Comment by Zeikko** [comment]
Yes we do handle this from our side and it is working fine. Let's see if someone else needs this as well.

**Comment by tomzaku** [comment]
Thank you. Please don't close this issue.
